checks/check_extra7177

```diff
@@ -0,0 +1,124 @@
+                                continue
+                            fi
+                            # Store additional master node security groups that allows access from the internet
+                            additional_master_node_sg_internet_open_list+=( "${additional_master_node_sg_internet_open}" )
+                        done
+                    fi
+
+                    # Retrive EMR Additional Slave node Security Groups rules
+                    additional_slave_node_sg_list=$("${AWSCLI}" emr describe-cluster --cluster-id "${cluster_id}" ${PROFILE_OPT} --region "${regx}" --query 'Cluster.Ec2InstanceAttributes.AdditionalSlaveSecurityGroups' --output text 2>&1)
+                    if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "${slave_node_sg}"; then
+                        textInfo "${regx}: Access Denied trying to describe EMR cluster" "${regx}" "${cluster_id}"
+                        continue
+                    fi
+                    local additional_slave_node_sg_internet_open_list
+                    if [[ "${additional_slave_node_sg_list}" != "None" ]]; then
+                        for additional_slave_node_sg in ${additional_master_node_sg_list}; do
+                            additional_slave_node_sg_internet_open=$("${AWSCLI}" ec2 describe-security-groups --group-ids "${additional_slave_node_sg}" --query 'SecurityGroups[?length(IpPermissions[?(contains(IpRanges[].CidrIp, `0.0.0.0/0`) || contains(Ipv6Ranges[].CidrIpv6, `::/0`))]) > `0`].{GroupId:GroupId}' ${PROFILE_OPT} --region "${regx}" --output text 2>&1)
+                            if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "${additional_slave_node_sg_internet_open}"; then
+                                textInfo "$regx: Access Denied trying to describe security groups" "$regx"
+                                continue
+                            fi
+                            # Store additional slave node security groups that allows access from the internet
+                            additional_slave_node_sg_internet_open_list+=( "${additional_slave_node_sg_internet_open}" )
+                        done
+                    fi
+
+                    # Check if EMR Cluster is publicly accessible through a Security Group
+                    if [[ -n "${master_node_sg_internet_open}" || -n "${slave_node_sg_internet_open}" || "${#additional_master_node_sg_internet_open_list[@]}" -ne 0 || "${#additional_slave_node_sg_internet_open_list[@]}" -ne 0 ]]; then
+                        textFail "${regx}: EMR Cluster ${cluster_id} is publicly accessible through the following Security Groups: Master Node ${master_node_sg_internet_open} ${additional_master_node_sg_internet_open_list[*]} -- Slaves Nodes ${slave_node_sg_internet_open} ${additional_slave_node_sg_internet_open_list[*]}" "${regx}" "${cluster_id}"
```

Comment:
Getting this error
```
An error occurred (InvalidGroupId.Malformed) when calling the DescribeSecurityGroups operation: Invalid id: "
An error occurred (InvalidRequestException) when calling the ListInstanceGroups operation: Instance fleets and instance groups are mutually exclusive. The EMR cluster specified in the request uses instance fleets. The ListInstanceGroups operation does not support clusters that use instance fleets. Use the ListInstanceFleets operation instead." (expecting "sg-...")
```

Reply:
Please, open a new Issue and paste the debug execution appending `bash -x`